static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -160,14 +182,91 @@ define([
                     trackDisplay: true,
                     cssModifierClass: colourClass
                 }).show(renderedBanner);
+
+            if(params.secondaryButtonCaption &&  SECONDARY_BUTTON != null) {
+                setSecondaryButtonListener();
+            }
+
             if (messageShown) {
                 mediator.emit('membership-message:display');
             }
             mediator.emit('banner-message:complete');
         }
 
+        function emailIsValid(email){
```

Comment:
This probably makes me sound like a lunatic, but can we make sure there's a space between the `)` and `{` for all function statements? :1st_place_medal: